src/components/icon/assets/view_charts.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="15" viewBox="0 0 16 15" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is the svg exported along with it's container from Figma?

Reply:
Yes indeed